code/game/objects/structures/bedsheet_bin.dm

```diff
@@ -271,22 +280,24 @@ LINEN BINS
 	extinguish()
 	update_icon()
 
-/obj/structure/bedsheetbin/attackby(obj/item/I as obj, mob/user as mob, params)
+/obj/structure/bedsheetbin/attackby(obj/item/I, mob/user, params)
 	if(istype(I, /obj/item/bedsheet))
 		user.drop_item()
-		I.loc = src
+		I.forceMove(src)
 		sheets.Add(I)
 		amount++
 		to_chat(user, "<span class='notice'>You put [I] in [src].</span>")
 	else if(amount && !hidden && I.w_class < WEIGHT_CLASS_BULKY)	//make sure there's sheets to hide it among, make sure nothing else is hidden in there.
+		if(I.flags & ABSTRACT)
+			return
 		user.drop_item()
```

Comment:
would want to test if this succeeds. For instance anti drop will fail (return FALSE) here

Reply:
Ah true. Will do later.